package.json

```diff
@@ -1,17 +1,18 @@
 {
   "name": "@raha.io/website",
-  "version": "0.2.0",
+  "version": "0.3.0",
   "scripts": {
-    "start": "react-scripts-ts start",
-    "build": "react-scripts-ts build",
-    "test": "react-scripts-ts test --env=jsdom",
+    "start": "yarn pre-build && react-scripts-ts start",
+    "build": "yarn pre-build && react-scripts-ts build",
+    "test": "yarn pre-build && react-scripts-ts test --env=jsdom",
+    "pre-build": "cp -r static public && static-i18n -o public -l en locales",
```

Comment:
A lot of this pre-build/post-build crap could be cleaned up if we run eject and then do things ourselves. Also need to make yarn start have hot-reload, I developed this by doing `serve -s locales` which was pretty lame.

However, not sure it makes sense to run eject any time soon given our mobile-first focus. At least wait for a couple upgrade cycles, might be nice to get whatever benefits come out of `react-scripts-ts 3.0`